### DISCUSSION TITLE
Not loading Android browser

### DISCUSSION THREAD

**Discussion by Arumugamhns** [google/model-viewer#2518] [Q&A]
Suddenly the below links are not working in the android mobile chrome browser but it's working fine in the desktop chrome browser
https://audi-india-ar.s3.amazonaws.com/A4/index.html 

is there any update need to implement here? 

and it seems its takes a very very long time for loading.

kindly give the suggestion for resolve the issue

**Comment by ghost** [comment]
I can only guess that something is going on with the hosting service, no
other reports like this. I'd have to see the implementation to make any
comment there, but there have been no breaking updates.

**Comment by JuanRenoldi** [comment]
I have the same problem with a complex model, but as far as I could test, this occurs on chrome for android. Samsung browser and safari can load the model just fine.

**Comment by elalish** [comment]
That site seems to be working on my pixel 3 and pixel 4. Can you fill out the rest of the issue template so that we can try to repro? And can you be very specific what you mean by "not working"? A screenshot would be nice.

**Comment by Arumugamhns** [comment]
> That site seems to be working on my pixel 3 and pixel 4. Can you fill out the rest of the issue template so that we can try to repro? And can you be very specific what you mean by "not working"? A screenshot would be nice.

Please find the screenshot
![image](https://user-images.githubusercontent.com/60291301/123375596-aab1d280-d5a6-11eb-8ad8-684cdb86d1e4.png)

**Comment by elalish** [comment]
What device? What operating system and version? What browser and version? Are there any console warnings or errors? Considering nothing is loading on the whole page, it may be something prior to model-viewer. Also, I noticed that the GLB is huge; you could be simply running out of memory. I would highly recommend optimizing it for mobile with smaller textures and fewer vertices.

**Comment by Arumugamhns** [comment]
> What device? What operating system and version? What browser and version? Are there any console warnings or errors? Considering nothing is loading on the whole page, it may be something prior to model-viewer. Also, I noticed that the GLB is huge; you could be simply running out of memory. I would highly recommend optimizing it for mobile with smaller textures and fewer vertices.

there is no error in console. android 11 mobile, chrome browser 91

**Comment by elalish** [comment]
I'm seeing a console error: `Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry': the name "model-viewer" has already been used with this registry
    at https://unpkg.com/@google/model-viewer@0.10.0/dist/model-viewer.js:63508:16`

It looks like you're loading both v0.10 and v1.7 of model-viewer. That seems likely to be causing problems.
